src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -2160,5 +2178,115 @@ public void run() {
 			}
 		};
 	}*/
-
+	
```

Comment:
Rather than duplicating code (Bad practice) can we instead change it so that this code is in one place and one place only (Perhaps an init() method) and called where needed instead? (I know this plugin already has bits of duplicated code, but it's not a good pattern to continue)

Reply:
Yeah I hated to do that but for some reason I couldn't think of something like init()... I really don't know why